src/Arcus.Templates.Tests.Integration/Fixture/TestConfig.cs

```diff
@@ -12,38 +14,89 @@ public class TestConfig : IConfigurationRoot
     {
         private readonly IConfigurationRoot _configuration;
 
-        private TestConfig(IConfigurationRoot configuration)
+        private TestConfig(IConfigurationRoot configuration, string buildConfiguration)
         {
             Guard.NotNull(configuration, nameof(configuration));
+            Guard.NotNullOrWhitespace(buildConfiguration, nameof(buildConfiguration));
 
             _configuration = configuration;
+            BuildConfiguration = buildConfiguration;
         }
 
+        /// <summary>
+        /// Gets the build configuration for the project created from the template.
+        /// </summary>
+        public string BuildConfiguration { get; }
+
         /// <summary>
         /// Creates a new <see cref="IConfigurationRoot"/> with test values.
         /// </summary>
-        public static TestConfig Create()
+        /// <param name="buildConfiguration">The configuration in which the created project from the template should be build.</param>
+        public static TestConfig Create(string buildConfiguration = "Debug")
         {
             var configuration = new ConfigurationBuilder()
                 .AddJsonFile(path: "appsettings.json", optional: true)
                 .AddJsonFile(path: "appsettings.local.json", optional: true)
                 .AddEnvironmentVariables()
                 .Build();
 
-            return new TestConfig(configuration);
+            return new TestConfig(configuration, buildConfiguration);
         }
 
         /// <summary>
-        /// Gets the base URL of the to-be-tested API.
+        /// Gets the project file path of the web API project.
+        /// </summary>
+        public DirectoryInfo GetWebApiProjectDirectory()
+        {
+            const string webApiProjectPath = "Arcus:Api:ProjectPath";
+
+            string projectPath = _configuration.GetValue<string>(webApiProjectPath);
+            Guard.NotNull(projectPath, nameof(projectPath), $"No project path configured for the web API project with the key: {webApiProjectPath}");
+            Guard.For<ArgumentException>(
+                () => !Directory.Exists(projectPath),
+                $"No project template directory exists at {projectPath}");
+
+            return new DirectoryInfo(projectPath);
+        }
+
+        /// <summary>
+        /// Gets the base URL of the to-be-created project from the web API template.
+        /// </summary>
+        /// <returns></returns>
+        public Uri GetDockerBaseUrl()
+        {
+            const string baseUrlKey = "Arcus:Api:BaseUrl";
+
+            string baseUrl = _configuration.GetValue<string>(baseUrlKey);
```

Comment:
A lot of this method is duplicated in `CreateWebAPiBaseUrl`, can we refactor it out?

Reply:
Yep. Thanks.